create-reporter/index.test.js

```diff
@@ -28,31 +28,34 @@ function clean (str) {
     .replace(/\d{4}-\d\d-\d\d \d\d:\d\d:\d\d/g, '1970-01-01 00:00:00')
     .replace(/"time":"[^"]+"/g, '"time":"1970-01-01T00:00:00.000Z"')
     .replace(/"hostname":"[^"]+"/g, '"hostname":"localhost"')
+    .replace(/"pid":\d+/g, '"pid":21384')
+    .replace(/PID:(\s+.*m)\d+(.*m)/, 'PID:$121384$2')
 }
 
 function check (type, details) {
   let json = new MemoryStream()
+  // TODO: why we need v property, it present in several examples in
+  //  pino docs, but there is no description what it is
   let jsonReporter = createReporter({
-    bunyan: bunyan.createLogger({ name: 'test', pid: 21384, stream: json })
+    logger: pino({
+      name: 'test',
```

Comment:
When we specify `pid`, we must specify hostname, otherwise it's undefined and missing from logs